src/Integration/Binding/CSharpVBBindingConfigProvider.cs

```diff
@@ -119,7 +119,15 @@ private async Task<IBindingConfig> DoGetConfigurationAsync(SonarQubeQualityProfi
                 bindingConfiguration.Project.ProjectKey,
                 language.FileSuffixAndExtension);
 
-            return new CSharpVBBindingConfig(ToVsRuleset(coreRuleset), ruleSetFilePath);
+            var additionalFilePath = solutionBindingFilePathGenerator.Generate(
+                bindingConfiguration.BindingConfigDirectory,
+                bindingConfiguration.Project.ProjectKey,
+                "params_" + language.FileSuffixAndExtension);
+
+            var ruleset = new FilePathAndContent<VsRuleset>(ruleSetFilePath, ToVsRuleset(coreRuleset));
+            var additionalFile = new FilePathAndContent<string>(additionalFilePath, "todo");
```

Comment:
@duncanp-sonar which API should I call to get the additional file content?
It used to be GetRoslynExportProfileAsync, which returned ConfigurationResponse, that had AdditionalFileResponse.

Reply:
```
var config = SonarLintConfigGenerator.Generate(activeRules, sonarProperties, language.ServerLanguage.Key);
```

... although the class is currently `internal` and `static` in `Core`, so it will need an interface for testing.